repository/src/main/java/org/apache/atlas/repository/store/graph/v1/DeleteHandlerV1.java

```diff
@@ -515,6 +516,23 @@ public void authorizeRemoveRelation(AtlasEdge edge) throws AtlasBaseException {
         RequestContext.get().endMetricRecord(metric);
     }
 
+    public void resetHasLineage(AtlasEdge edge) throws AtlasBaseException {
```

Comment:
Nixon, resetting will be handled in https://github.com/atlanhq/atlas-metastore/pull/366/files
This is live on beta, please get this change & check entity delete scenario
Also, I realized that I missed delete relationship API flow there, let us fix that in https://github.com/atlanhq/atlas-metastore/pull/366/files itself

Reply:
@nikhilbonte21 , I merged the _removeHasLineageOnDelete_ method  for lineage entity delete into this PR and tested with it. Please review PR once.